reference/5.1/Microsoft.PowerShell.Core/About/about_History.md

```diff
@@ -41,6 +41,9 @@ command history.
   <kbd>F8</kbd>. Press <kbd>F8</kbd> again the next instance.
 - <kbd>F9</kbd> - Find a command by history ID. Type the history ID then press
   <kbd>F9</kbd>. Press <kbd>F7</kbd> to find the ID.
+- <kbd>#</kbd>`<string>`</kbd><kbd>Tab</kbd> - search the history for
```

Comment:
S in search should be capitalized for consistency.

Reply:
Fixed